### DISCUSSION TITLE
div

### DISCUSSION THREAD

**Discussion by MohdZabih** [akash-coded/mern#33] [Q&A]
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>HTML Div Tag Demo</title>
  </head>

  <body>
    
    <div><h1>HTML Div Tag</h1></div>

    
    <div>
      <div>
        <h2>Unordered Lists</h2>
        
        <div>
          Unordered List
          <ul>
            <li>List Item-1</li>
            <li>List Item-2</li>
            <li>...</li>
          </ul>
        </div>

        
        <div>
          Unordered List with Square Bulleting
          <ul style="list-style-type: square">
            <li>List Item-1</li>
            <li>List Item-2</li>
            <li>...</li>
          </ul>
        </div>

        
        <div>
          Unordered List with Disc Bulleting
          <ul style="list-style-type: disc">
            <li>List Item-1</li>
            <li>List Item-2</li>
            <li>...</li>
          </ul>
        </div>

        
        <div>
          Unordered List with Circle Bulleting
          <ul style="list-style-type: circle">
            <li>List Item-1</li>
            <li>List Item-2</li>
            <li>...</li>
          </ul>
        </div>
      </div>

      <div>
        <h2>Ordered Lists</h2>
        Ordered List
        <ol>
          <li>List Item-1</li>
          <li>List Item-2</li>
          <li>...</li>
        </ol>

        Ordered List with Alphabetical Enumeration
        <ol type="A">
          <li>List Item-1</li>
          <li>List Item-2</li>
          <li>...</li>
        </ol>

        Ordered List with alphabetical Enumeration
        <ol type="a">
          <li>List Item-1</li>
          <li>List Item-2</li>
          <li>...</li>
        </ol>

        Ordered List with Roman Numerals
        <ol type="I">
          <li>List Item-1</li>
          <li>List Item-2</li>
          <li>...</li>
        </ol>

        Ordered List with roman numerals
        <ol type="i">
          <li>List Item-1</li>
          <li>List Item-2</li>
          <li>...</li>
        </ol>
      </div>

      <div>
        <h2>Nested Lists</h2>
        Nested Unordered Lists
        <ul>
          <li>List Item-1</li>
          <li>
            List Item-2
            <ul>
              <li>Sublist Item-1</li>
              <li>Sublist Item-2</li>
            </ul>
          </li>
          <li>List Item-3</li>
        </ul>

        Nested Ordered Lists
        <ol>
          <li>List Item-1</li>
          <li>
            List Item-2
            <ol type="i">
              <li>Sublist Item-1</li>
              <li>Sublist Item-2</li>
            </ol>
          </li>
          <li>List Item-3</li>
        </ol>

        Nested Mixed Lists
        <ol>
          <li>List Item-1</li>
          <li>
            List Item-2
            <ul>
              <li>Sublist Item-1</li>
              <li>Sublist Item-2</li>
            </ul>
          </li>
          <li>
            List Item-3
            <ol type="i">
              <li>Sublist Item-1</li>
              <li>Sublist Item-2</li>
            </ol>
          </li>
          <li>List Item-4</li>
        </ol>
      </div>
    </div>
  </body>
</html>

**Comment by Egor-S-N** [comment]
html?)
